internal/node/startup/startup.go

```diff
@@ -22,7 +22,7 @@ import (
 // Validates the Node Database Schema Version
 func ValidateSchema(config config.NodeConfig) error {
 	endpoint := config.PostgresEndpoint.Value
-	if config.PostgresSslMode {
+	if !config.PostgresSslMode {
```

Comment:
Our current policy is to have the configs always as "enable feature X"  to avoid user confusion. We used to have a mixture of "feature A enable" and "feature B disable" properties.

Reply:
@marcelstanley  I don't understand. We already normalized. Everything is SomethingEnabled, as @fmoura pointed out.

I don't mind changing the NodeConfig field to *disable* so much, but I don't see the benefit.